posthog/session_recordings/session_recording_api.py

```diff
@@ -37,12 +37,9 @@
     SessionRecordingViewed,
 )
 
-from posthog.session_recordings.queries.session_recording_list_from_replay_summary import (
-    SessionRecordingListFromReplaySummary,
-    SessionIdEventsQuery,
-)
 from posthog.session_recordings.queries.session_recording_list_from_filters import (
     SessionRecordingListFromFilters,
+    EventsSubQuery,
```

Comment:
Wonder if we need a more descriptive name here because it's exported outside of the class. Maybe `SessionRecordingEventsSubQuery`?